src/test/java/org/apache/maven/plugins/shade/mojo/ShadeMojoTest.java

```diff
@@ -62,6 +62,43 @@
 public class ShadeMojoTest
     extends PlexusTestCase
 {
+    public void testSkipShading() throws Exception
+    {
+        final ShadeMojo mojo = new ShadeMojo();
```

Comment:
This is not a not a PlexusTest, but a unittest using reflection. And I understand it follows other test examples, but this keeps increasing the technical dept.
I suggest to drop this as the non-default value is already covered by the integration test, the default value with all the other tests.

Reply:
We dropped this meanwhile.